core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/QualificationBenchmark.scala

```diff
@@ -0,0 +1,27 @@
+package org.apache.spark.rapids.tool.benchmarks
+
+import com.nvidia.spark.rapids.tool.qualification.QualificationArgs
+import com.nvidia.spark.rapids.tool.qualification.QualificationMain.mainInternal
+
+object QualificationBenchmark extends BenchmarkBase {
+  override def runBenchmarkSuite(iterations: Int,
+    warmUpIterations: Int,
+    outputFormat: String,
+    mainArgs: Array[String]): Unit = {
+    runBenchmark("QualificationBenchmark") {
+      val benchmarker =
+        new Benchmark(
+          "QualificationBenchmark",
+          2,
+          output = output,
+          outputPerIteration = true,
+          warmUpIterations = warmUpIterations,
+          minNumIters = iterations)
+      benchmarker.addCase("QualificationBenchmark") { _ =>
+        mainInternal(new QualificationArgs(mainArgs),
```

Comment:
I am little bit confused here. If we initialize `QualificationArgs` on line 21, then what are those default arguments that are passed to the `benchmarker` variable?

Reply:
Clarified post discussion. Benchmarker arguments control iterations and warm up iterations. The passed ffunction to the benchmarker is what is benchmarked internally

Reply:
We will iterate on this later.